nodes/number/float_math.py

```diff
@@ -58,6 +66,9 @@ def create(self):
 
     def draw(self, layout):
         layout.prop(self, "operation", text = "")
+        
+    def draw_label(self):
```

Comment:
uhh you're right here. In fact I didn't know about this function. We can leave it as it is now but most propably I will add here an abstraction layer as well. 
Then we will have the functions: 'draw', 'drawAdvanced' and 'drawLabel'